tools/upload_text.php

```diff
@@ -270,6 +295,15 @@
             notify_project_event_subscribers( $project, 'sr_reported' );
         }
 
+        if (($stage == "in_prog_1") || ($stage == "in_prog_2"))
+        {
+            // record postcomments in projects table
```

Comment:
We're trying to standardize on the following format:
```php
$sql = sprintf("
    UPDATE projects
    SET postcomments = CONCAT(postcomments, '%s')
    WHERE projectid = '%s'
", DPDatabase::escape($postcomments),
    DPDatabase::escape($projectid));
DPDatabase::query($sql);
```

See the updated [coding best practices](https://www.pgdp.net/wiki/DP_Code_Best_Practices#Escaping_in_SQL).

Reply:
Done this.